controller-norbac.jsonnet

```diff
@@ -32,6 +32,8 @@ local controllerContainer =
   container.command(["controller"]) +
   container.livenessProbe(controllerProbe) +
   container.readinessProbe(controllerProbe) +
+  container.mixin.securityContext.readOnlyRootFilesystem(true) +
+  container.mixin.securityContext.runAsUser(1001) +
```

Comment:
Do you know why I couldn't reproduce this locally? This is the command I run to get the .ymls

```bash
$ kubecfg show -o yaml controller-norbac.jsonnet -J /Users/tompizmor/Desktop/github-repos/ksonnet-lib-beta.1
```